sql/amo_dev/amo_stats_dau_v2/metadata.yaml

```
@@ -0,0 +1,11 @@
+friendly_name: AMO Stats DAU dev/stage
+description: >
```

Nit: To prevent trialing newlines in case this description shows up in BigQuery
```suggestion
description: >-
```